src/lib/OverlayView.js

```diff
@@ -104,6 +114,18 @@ export default _.flowRight(
     this._containerElement = helpers.createContainerElement();
   },
 
+  // MarkerClusterer needs getDraggable method,
+  // but since OverlayView is not draggable by default
+  // we just return false
+  getDraggable() {
+    return false;
+  },
+
+  // MarkerClusterer needs maps LatLng object
+  getPosition() {
+    return new google.maps.LatLng(this.props.position);
```

Comment:
I'm wondering if someone changes the `props.position`, will the new position get propagated to the `MarkerClusterer` instance?